other/fisherman/Cargo.toml

```diff
@@ -0,0 +1,21 @@
+[package]
+name = "fisherman"
+version = "0.1.0"
+edition = "2018"
+
+[dependencies]
+sbor = { path = "/Users/woof/Developing/Projects/Blockchain/radix/radixdlt-scrypto/sbor" }
```

Comment:
Can you make this example compatible with scrypto v0.2.0 please ?

Reply:
@cbisaillon 

Done

Thanks!